backend/amplify/backend/function/scrubFunc/src/app.js

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
+    http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+*/
+
+var express = require("express");
+var bodyParser = require("body-parser");
+var sharp = require("sharp");
+var awsServerlessExpressMiddleware = require(
```

Comment:
ES6 supports some new (better?) ways to do require. `var` is fine for hackathons.
https://www.freecodecamp.org/news/requiring-modules-in-node-js-everything-you-need-to-know-e7fbd119be8/